components/tabs/src/fixtures.js

```diff
@@ -340,4 +342,60 @@ HooksExample.defaultProps = sharedDefaultProps;
 
 HooksExample.propTypes = sharedPropTypes;
 
-export { HooksExample, ProposedClassPropertiesPlugin, SimpleTabs, SimpleMapTabs, TableTabs };
+// This example shows all panels when in mobile mode, but the route will still be updated
+// to match the last clicked tab title.
+// To show only the active panel on mobiles, simply remove the `isTablet` code.
+// eslint-disable-next-line react/prop-types
+const RouterTabs = ({ match: { params: { section } } }) => {
+  const isTablet = useMedia(`(min-width: ${BREAKPOINTS.TABLET})`);
```

Comment:
- React components should be pure functions where possible
- I want govuk-react to be usable with universal and server-only React
- I want our fixtures to be representative of real world use cases, they can be simplified but would prefer the examples work universal or server only unless there's a clear reason why this would be a client side only thing

however

- useMedia cannot be used on the server AFAIK
- useMedia means the component is not a pure function AFAIK

so

- if we have to show a client side only solution can we mark it as such and ensure there is an appropriate alternative for server side
- can we just have an example without useMedia? as per your comment "To show only the active panel on mobiles, simply remove the `isTablet` code"

Reply:
in general I'd prefer we don't introduce a dev dependency on something I consider to be an anti pattern (react-use-media) due to it conflicting with universal rendering and pure functions

Reply:
As per my other comment below, I've adjusted the comments and name for this example, and added two new examples.

I completely understand where you're coming from.  The principle that govuk-react should support SSR is very important.

This particular example is client-side only.  The component inherently needs to know how wide the screen is, and to adjust when it changes.  It should thus be documented accordingly.

I'm not sure I entirely agree that this component is not a "pure function".  Given the same inputs (props and window size), this component should always render the same output.

The grey area comes in given the fact that when the media query listener created by `useMedia` sees a state change, it will then trigger a state update on the component.  That however is the point of effect hooks - to allow for state changes in pure functions.

Reply:
> I'm not sure I entirely agree that this component is not a "pure function". Given the same inputs (props and window size), this component should always render the same output.

window size is not an input to the function